src/core_ocean/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -372,6 +378,15 @@ function ocn_forward_mode_init(domain, startTimeStamp) result(ierr)!{{{
 
       endif
 
+
+      if (config_debug_split_explicit) then
+        ! open up temp file
+        call mpas_new_unit(debugSplitExplicitUnit)
+        ! note, presumes we are running on a single core here...
+        open(unit=debugSplitExplicitUnit, file=trim(config_debug_split_explicit_filename), &
+          action='write', status='unknown')
+      end if
+
```

Comment:
@mark-petersen, this was the fix we discussed previously as mentioned in your email.